Formula/tbb.rb

```diff
@@ -126,7 +152,7 @@ def install
 +        install --prefix ${PYTHON_BUILD_WORK_DIR}/build --install-lib ${PYTHON_BUILD_WORK_DIR}/build/@@SITE_PACKAGES@@ -f
      COMMENT "Build and install to work directory the oneTBB Python module"
  )
- 
+
```

Comment:
Let's not touch whitespace in the embedded patch, please. (Yes, I know it was probably your editor that ate this.)